package.json

```diff
@@ -15,21 +15,23 @@
     "banners": "./utils/maintain-banners.js",
     "prebuild": "rimraf ./src/models ./src/factories ./src/generated-client.js",
     "build": "okta-sdk-generator -t templates/ -o .",
-    "eslint": "eslint -c src/.eslintrc src/ && eslint -c test/.eslintrc test/it test/unit && eslint -c test/jest/.eslintrc test/jest/ && eslint -c src/.eslintrc utils/",
+    "eslint": "eslint -c src/.eslintrc src/ --ignore-pattern 'src/types/' && eslint -c test/.eslintrc test/it test/unit && eslint -c test/jest/.eslintrc test/jest/ && eslint -c src/.eslintrc utils/ && eslint -c templates/.eslintrc templates && eslint -c src/types/.eslintrc src/types",
```

Comment:
How about refactoring the lint rule to have the common rules sit in the root directory, then specific rules in each separated dirs? Then the lint command can be changed to `eslint .`.